projection/core/src/main/scala/com/lightbend/lagom/internal/projection/ProjectionRegistryActor.scala

```diff
@@ -68,24 +70,32 @@ class ProjectionRegistryActor extends Actor with ActorLogging {
   val writeConsistency: WriteConsistency = WriteMajority(timeout = 5.seconds)
 
   // (a) Replicator contains data of all workers (requested and observed status, plus a name index)
-  private val RequestedStatusDataKey: LWWMapKey[WorkerKey, Status] =
-    LWWMapKey[WorkerKey, Status]("projection-registry-requested-status")
+  private val WorkerRequestedStatusDataKey: LWWMapKey[WorkerKey, Status] =
+    LWWMapKey[WorkerKey, Status]("worker-level-requested-status")
+  private val ProjectionRequestedStatusDataKey: LWWMapKey[ProjectionName, Status] =
+    LWWMapKey[ProjectionName, Status]("projection-level-requested-status")
```

Comment:
This PR introduces the new `ProjectionRequestedStatusDataKey` which keeps track of projection-level  intentions. Then, the whole `ProjectionRegistryActor` is responsible for merging default, projection-level and worker-level intentions into actual desired statuses.